ui/src/components/chat/sidebar.rs

```diff
@@ -161,9 +172,18 @@ pub fn Sidebar(cx: Scope<Props>) -> Element {
                     let without_me = state.read().get_without_me(chat.participants.clone());
                     let user = without_me.first();
                     let default_message = Message::default();
-                    let parsed_user = match user {
-                        Some(u) => u.clone(),
-                        None => Identity::default(),
+                    let parsed_user = user.cloned().unwrap_or_default();
+
+                    let platform = match parsed_user.platform() {
+                        warp::multipass::identity::Platform::Desktop => Platform::Desktop,
```

Comment:
Side note: These could be done in a single function so there isnt alot of match condition, but I think changes might be done later so this conversion wouldnt be needed since warp didint originally have `Platform` or the additional status variants in its enum.